service/awsconfig/v4/resource/cloudformation/adapter/host_iam_roles.go

```diff
@@ -2,7 +2,10 @@ package adapter
 
 import "github.com/giantswarm/aws-operator/service/awsconfig/v4/key"
 
-// template related to this adapter: service/templates/cloudformation/host-pre/iam_roles.yaml
+// The template related to this adapter can be found in the following import.
+//
+//     github.com/giantswarm/aws-operator/service/awsconfig/v4/templates/cloudformation/hostpre/iam_roles.go
+//
```

Comment:
Just changed the comments here according to the new location. This should actually be rewritten when bumping versions. The script @kopiczko provided should also help here.

Reply:
Nice if the script updates this. The comments also become less important now the templates are under v4.